lib/install.js

```diff
@@ -39,8 +39,15 @@ function install (gyp, argv, callback) {
     }
   }
 
-  var distUrl = gyp.opts['dist-url'] || gyp.opts.disturl || 'https://nodejs.org/dist'
+  var distUrl = gyp.opts['dist-url'] || gyp.opts.disturl
 
+  if (!distUrl) {
+    if (semver.satisfies(version, '>=1.0.0')) {
```

Comment:
This may not reliable if NodeJS released `v1.0.0` someday, I prefer `process.release.node === 'io.js'`.